playbooks/robusta_playbooks/babysitter.py

```diff
@@ -63,11 +65,7 @@ def resource_babysitter(event: KubernetesAnyChangeEvent, config: BabysitterConfi
         finding_type=FindingType.CONF_CHANGE,
         failure=False,
         aggregation_key=f"ConfigurationChange/KubernetesResource/Change",
-        subject=FindingSubject(
-            event.obj.metadata.name,
-            FindingSubjectType.from_kind(event.obj.kind),
-            event.obj.metadata.namespace,
-        ),
+        subject=KubeObjFindingSubject(event.obj),
```

Comment:
I think in configuration change, we would like to connect it to the node, only if the change is on the node